CodeJam.Main/Strings/StringBuilderExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Text;
+
+using JetBrains.Annotations;
+
+namespace CodeJam.Strings
+{
+	/// <summary>
+	/// <see cref="StringBuilder"/> class extensions.
+	/// </summary>
+	[PublicAPI]
+	public static class StringBuilderExtensions
+	{
+		/// <summary>
+		/// Determines whether the StringBuilder is null or empty.
+		/// </summary>
+		/// <param name="value">The StringBuilder to test.</param>
+		/// <returns>
+		/// <c>true</c> if <paramref name="value"/> is null or empty; otherwise, <c>false</c>.
+		/// </returns>
+		[Pure]
+		public static bool IsNullOrEmpty([CanBeNull] this StringBuilder value) =>
+			value == null || value.Length == 0;
+
+		/// <summary>
+		/// Determines whether the StringBuilder is netier null nor empty.
```

Comment:
Typo: netier -> neither

Reply:
Thanks.
Seems like my spell checker was not configured to check comments :(